spec/services/idv/profile_maker_spec.rb

```diff
@@ -135,28 +136,70 @@
     end
 
     context 'with in_person_verification_needed' do
-      let(:profile) do
-        subject.save_profile(
-          fraud_pending_reason: nil,
-          gpo_verification_needed: false,
-          deactivation_reason: nil,
-          in_person_verification_needed: true,
-          selfie_check_performed: false,
-        )
-      end
-      it 'creates a pending profile for in person verification' do
-        expect(profile.activated_at).to be_nil
-        expect(profile.active).to eq(false)
-        expect(profile.deactivation_reason).to be_nil
-        expect(profile.fraud_pending_reason).to be_nil
-        expect(profile.in_person_verification_pending?).to eq(true)
-        expect(profile.fraud_review_pending?).to eq(false)
-        expect(profile.gpo_verification_pending_at.present?).to eq(false)
-        expect(profile.initiating_service_provider).to eq(nil)
-        expect(profile.verified_at).to be_nil
+      context 'when threatmetrix is disabled' do
```

Comment:
```suggestion
      context 'when threatmetrix decisioning is disabled' do
```

If you don't mind, I want to try to be more consistent in communicating that TMX is running either way, and this is just whether we're going to act based on its value.